flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -331,6 +331,22 @@
                                     + "JobManager could be faster, since no reverse DNS lookup is performed. "
                                     + "However, local input split assignment (such as for HDFS files) may be impacted.");
 
+    @Documentation.Section(Documentation.Sections.EXPERT_JOB_MANAGER)
+    public static final ConfigOption<Integer> JOB_MANAGER_FUTURE_THREADS =
+            key("jobmanager.future-executor.numThreads")
```

Comment:
may be `pool-size` will be more accurate than `numThreads` ? which can be aligned to `CLUSTER_IO_EXECUTOR_POOL_SIZE` and same to the description.